### DISCUSSION TITLE
[Bug]: O campo maxValue não esta validando.

### DISCUSSION THREAD

**Discussion by joaonivaldo** [leandrocfe/filament-ptbr-form-fields#60] [Q&A]
### What happened?

Quando eu uso no Money ele não valida o valor que esta no maxValue ou dentro do rule.

### How to reproduce the bug

Money::make('TRATO1')
    ->suffix('%')
    ->prefix(null)
    ->minValue(0)
    ->maxValue(10)
    ->default('0')
    ->required()

ou 

Money::make('TRATO1')
    ->suffix('%')
    ->prefix(null)
    ->rules([ 'min:0' , 'max:10'])
    ->default('0')
    ->required()

### Package Version

3.1.0

### PHP Version

8.3

### Laravel Version

11.34.0

### Which operating systems does with happen with?

Windows, Linux

**Comment by jlucfarias** [comment]
@joaonivaldo se você adicionar as duas linhas que eu coloquei [aqui](https://github.com/leandrocfe/filament-ptbr-form-fields/issues/13#issuecomment-2881118725), provavelmente vá resolver a questão (ao menos enquanto não é adicionado uma forma nativa de fazer isso)
